src/components/augmentedReality/AugmentedReality.js

```diff
@@ -58,40 +58,38 @@ export const AugmentedReality = ({ navigation, onSettingsScreen }) => {
 
   return (
     <>
-      <WrapperWithOrientation>
-        <Wrapper>
-          <Touchable
-            onPress={() => navigation.navigate(ScreenName.ARInfo, { data, isLoading, refetch })}
-          >
-            <WrapperRow spaceBetween>
-              <RegularText>{texts.augmentedReality.whatIsAugmentedReality}</RegularText>
-              <Icon.ArrowRight size={normalize(20)} />
-            </WrapperRow>
-          </Touchable>
-        </Wrapper>
-        <Wrapper>
-          <Button
-            onPress={() => setIsModalVisible(!isModalVisible)}
-            invert
-            title={texts.augmentedReality.loadingArtworks}
-          />
-        </Wrapper>
-        <TitleContainer>
-          <Title accessibilityLabel={`(${texts.augmentedReality.worksOfArt}) ${a11yText.heading}`}>
-            {texts.augmentedReality.worksOfArt}
-          </Title>
-        </TitleContainer>
-        {device.platform === 'ios' && <TitleShadow />}
-
-        <ARObjectList
-          data={data}
-          setData={setData}
-          isLoading={isLoading}
-          navigation={navigation}
-          refetch={refetch}
-          showOnDetailPage
+      <Wrapper>
+        <Touchable
+          onPress={() => navigation.navigate(ScreenName.ARInfo, { data, isLoading, refetch })}
+        >
+          <WrapperRow spaceBetween>
+            <RegularText>{texts.augmentedReality.whatIsAugmentedReality}</RegularText>
+            <Icon.ArrowRight size={normalize(20)} />
+          </WrapperRow>
+        </Touchable>
+      </Wrapper>
```

Comment:
this whole section is used here and in the same way in another file. this could be a component reused in both places to reduce duplication, right?

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/571/commits/f9fa9a2dbf12a08356e24d512e0e7e6e56657ff1